front/pages/w/[wId]/builder/assistants/[aId]/index.tsx

```diff
@@ -6,18 +6,24 @@ import AssistantBuilder, {
 import { getAgentConfiguration } from "@app/lib/api/assistant/configuration";
 import { getDataSources } from "@app/lib/api/data_sources";
 import { Authenticator, getSession, getUserFromSession } from "@app/lib/auth";
+import { ConnectorsAPI } from "@app/lib/connectors_api";
 import { isDevelopmentOrDustWorkspace } from "@app/lib/development";
 import { AgentConfigurationType } from "@app/types/assistant/agent";
 import { DataSourceType } from "@app/types/data_source";
 import { UserType, WorkspaceType } from "@app/types/user";
 
 const { GA_TRACKING_ID = "" } = process.env;
 
+type DataSourceConfig = NonNullable<
+  AssistantBuilderInitialState["dataSourceConfigs"]
```

Comment:
Can we rename to `dataSourceConfigurations` in `AssistantBuilderState`?

Reply:
Ok